src/illumidesk/spawners/spawners.py

```diff
@@ -1,27 +1,50 @@
 import os
-import shutil
 
 from dockerspawner import DockerSpawner
+from illumidesk.spawners.hooks import custom_auth_state_hook
+from illumidesk.spawners.hooks import custom_pre_spawn_hook
 
 
-class IllumiDeskRoleDockerSpawner(DockerSpawner):
+class IllumiDeskDockerSpawner(DockerSpawner):
+    """
+    Defines the common behavior for our Spwaners that work with LTI versions 1.1 and 1.3
```

Comment:
Change to """Extends the DockerSpawner ...""". Perhaps we need to rename this class to `IllumiDeskBaseDockerSpawner` for clarity?

Reply:
I agree